components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -166,7 +195,9 @@ function AutomaticPointsTable:generateReverseAliases(teams, tournaments)
 		reverseAliases[tournamentIndex] = {}
 		Table.iter.forEachIndexed(teams,
 			function(index, team)
-				local alias = mw.language.getContentLanguage():ucfirst(team.aliases[tournamentIndex])
+				local alias = mw.getContentLanguage():ucfirst(

```

Comment:
`ucfirst` shouldn't be needed with resolve redirect

Also I'd recommend making resolve redirects optional (on a wiki by wiki basis). Many wikis don't do resolve redirects.

Reply:
> `ucfirst` shouldn't be needed with resolve redirect
> 
> Also I'd recommend making resolve redirects optional (on a wiki by wiki basis). Many wikis don't do resolve redirects.

I think [this solution](https://github.com/Liquipedia/Lua-Modules/pull/1392#discussion_r901039986) satisfies